pkg/model/utxo/utxo.go

```diff
@@ -107,6 +105,11 @@ func (u *Manager) PruneMilestoneIndexWithoutLocking(msIndex milestone.Index, pru
 			mutations.Cancel()
 			return err
 		}
+
+		if err := deleteOutputLookups(spent.output, mutations); err != nil {
+			mutations.Cancel()
+			return err
+		}
```

Comment:
I don't get that part here.
The output lookups are the "unspent" lookups.... Why would we delete them?

Reply:
We are pruning the spent outputs from the db. This is just a cleanup. Normally there should be no lookups left since the output was already marked as spent, but better be thorough :-)